sandbox.html

```diff
@@ -221,12 +246,29 @@
   </div>
   <div>
     <label for="geometry">Geometry (GeoJSON) *</label>
-    <textarea name="geometry" id="geometry" rows="5" required></textarea>
+    <textarea name="geometry" id="geometry" rows="5" required>
+{
+  "type": "Point",
+  "coordinates": [8.682127, 50.1109238]
+}
+</textarea>
   </div>
   <div>
     <label for="minScale">Minimum Scale</label>
     <input type="number" name="minScale" id="minScale">
   </div>
+  <div>
+    <label for="useMapSrs">Use map SRS</label>
+    <input type="checkbox" name="useMapSrs" id="useMapSrs" />
+  </div>
+  <div>
+    <label for="mapExtent">Map extent coordinates</label>
+    <input name="mapExtent" id="mapExtent" placeholder="minx,miny,maxx,maxy" value="5.533762,47.304145,16.168528,54.190365">
```

Comment:
Would be good to use consistent casing for this placeholder. showMap uses minX,minY,maxX,maxY